com.unity.ml-agents/Tests/Editor/BehaviorParameterTests.cs

```diff
@@ -0,0 +1,29 @@
+using NUnit.Framework;
+using UnityEngine;
+using MLAgents;
+using MLAgents.Policies;
+
+namespace MLAgents.Tests
+{
+    [TestFixture]
+    public class BehaviorParameterTests
+    {
+        static float[] DummyHeuristic()
+        {
+            return null;
+        }
+
+        [Test]
+        public void TestNoModelInferenceOnlyThrows()
+        {
+            var gameObj = new GameObject();
```

Comment:
Not sure if there's a better file to put this in.

Reply:
I think it is fine.